Author: Jaromír Mikeš <mira.mikes@seznam.cz>
Origin: Debian
Description: Fix FTBS with compat 9. Thanks to Sebastian Ramacher <sramacher@debian.org>.
Forwarded: yes

Index: yoshimi/src/UI/MasterUI.fl
===================================================================
--- yoshimi.orig/src/UI/MasterUI.fl	2013-08-19 04:01:42.000000000 +0200
+++ yoshimi/src/UI/MasterUI.fl	2013-08-19 04:04:08.568969766 +0200
@@ -880,10 +880,10 @@
             if (result == -10)
             {
                 msg += ",\\nit does not appear to be a valid parameter file.";
-                fl_alert(msg.c_str());
+                fl_alert("%s", msg.c_str());
             }
             else
-                fl_alert(msg.c_str());
+                fl_alert("%s", msg.c_str());
         }} {}
   }
   Function {do_save_master(const char* file = NULL)} {} {
